src/Twig/AppExtension.php

```diff
@@ -11,16 +11,99 @@ public function getFunctions()
     {
         return [
             new TwigFunction('alpha', [$this, 'toAlpha']),
+            new TwigFunction('match_coordinate', [$this, 'matchCoordinate']),
+            new TwigFunction('to_array', [$this, 'toArray']),
+            new TwigFunction('grid_row_start', [$this, 'getGridRowStart']),
```

Comment:
You don't need all this "grid" twig functions IMO.
You'd better create a single function that return the right value for theses 4 functions

Reply:
You would return the entire attribute as a string ?

Reply:
Yes, you don't like this idea?

Reply:
Yes, I agree